content/rest/guides/scripting-with-the-rest-api-and-net.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Scripting with the REST API and .NET
```

Comment:
C# is not a scripting language without using an add-on like MSFT's C# REPL, the interactive window, or `dotnet-script`. Please change the language throughout this document and the link(s) to it, perhaps to something like "application" or "program".